kartothek/utils/predicate_converter.py

```diff
@@ -0,0 +1,88 @@
+"""
+Helper module to convert kartothek dataset load predicates into cube conditions.
+"""
+
+from typing import Any, List, Sequence, Tuple
+
+import pandas as pd
+import pyarrow as pa
+
+from kartothek.core.cube.conditions import Condition
+
+
+def write_predicate_as_cube_condition(predicate: Tuple[str, str, Any]) -> Condition:
+    """
+    Rewrites a single io.dask.dataset 'read_dataset_as_ddf' predicate condition as cube condition.
+
+    Remark: This function is restricted by "Condition.from_string" which does not allow for IsInCondition
+    and InIntervalCondition and will throw an error if conditions of those types are passed.
+
+    Parameters
+    ----------
+    predicate: list
+        list containing single predicate definition
+
+    Returns
+    -------
+    condition: cube condition object
```

Comment:
```suggestion
    condition: Condition
```

Reply:
@infehm this is ready to be merged now. Can you add a changelog entry for this change? I think the failing doc build can be fixed the suggestion

Reply:
I guess I also need to change it for the below function, right?  (convert_predicates_to_cube_conditions)